www/js/transforms/widenImages.js

```diff
@@ -28,17 +28,17 @@ function shouldWidenImage(image) {
         return false;
     }
     
-    // Some wide images are wrapped in a <div style="overflow-x:auto">...</div> so they can
-    // scroll side to side if needed without causing the entire section to scroll side to side.
-    // Such images don't need further widening.
-    if(image.hasAttribute('hasOverflowXContainer')){
+    // Some images are within a <div class="noresize">...</div> which indicates
+    // they should not be widened. Example below has links overlaying such an image.
+    // See:
+    //      "enwiki > Counties of England > Scope and structure > Local government"
+    if(utilities.findClosest(image, "[class*='noresize']")){
```

Comment:
@niedzielski If you remove the image overflow container transform from Android (before "widenImages" is consolidated) this is the line you'd have to change in "widenImages".